runtime/onert/core/src/compiler/ExecutorFactory.cc

```diff
@@ -39,6 +39,13 @@
 #include <functional>
 #include <memory>
 
+// Headers For training
```

Comment:
Do we need `ENABLE_ONERT_TRAIN` flag here?

Reply:
Yes, I will add it.